test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -75,49 +83,43 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
       playStatus(transferSummaryPage) mustBe OK
       contentType(transferSummaryPage) mustBe Some("text/html")
       contentAsString(transferSummaryPage) must include("transferSummary.header")
+
       contentAsString(transferSummaryPage) must include("transferSummary.seriesReference")
+      contentAsString(transferSummaryPage) must include(seriesCode.get.code.get)
```

Comment:
Is there a reason this can't just be something like `seriesCode.code` or `seriesCode.code.value` rather than `seriesCode.get.code.get`? More just wondering than anything else, same for the others below.

Reply:
Unfortunately, it has to be this way because the getConsignmentSummary expects an option (the series) and the series code within it is an option

Reply:
Ah I see! :+1: